v1/ao/internal/reporter/reporter_test.go

```diff
@@ -458,11 +300,11 @@ func TestShutdownGRPCReporter(t *testing.T) {
 	setGlobalReporter("ssl")
 
 	require.IsType(t, &grpcReporter{}, globalReporter)
-	time.Sleep(time.Second * 5)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Why sleeping after checking if globalReporter variable has an interface type of grpcReporter? The function `setGlobalReporter` is not going to return until the global reporter is created.

Reply:
good catch, removing that leftovers

Reply:
good catch, removing the sleeps